src/poliastro/twobody/orbit.py

```diff
@@ -385,15 +388,23 @@ def from_body_ephem(cls, body, epoch=None):
             epoch = epoch.tdb
             warn(
                 "Input time was converted to scale='tdb' with value "
-                "{}. Use Time(..., scale='tdb') instead.".format(epoch.tdb.value),
+                "{}. Use Time(..., scale='tdb') instead.".format(
+                    epoch.tdb.value),
                 TimeScaleWarning,
             )
-
-        r, v = get_body_barycentric_posvel(body.name, epoch)
-
+        try:
```

Comment:
Excellent! Now, please remove it from its original location :)

Reply:
I did, I didn't stage my previous commit properly. I apologize.